mssql-plugin/pom.xml

```diff
@@ -38,6 +38,11 @@
       <artifactId>database-commons</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>com.microsoft.azure</groupId>
+      <artifactId>adal4j</artifactId>
+      <version>1.6.4</version>
```

Comment:
did you check license? Can it be used?

Reply:
ADAL library uses MIT license so it is ok for us to use it.